code/modules/antagonists/heretic/knowledge/rust_lore.dm

```diff
@@ -265,10 +291,44 @@
 		sound = 'sound/ambience/antag/heretic/ascend_rust.ogg',
 		color_override = "pink",
 	)
-	new /datum/rust_spread(loc)
+	//new /datum/rust_spread(loc)
```

Comment:
Don't keep in commented out code

Likewise if `/datum/rust_spread` is unused now delet it